release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -2679,15 +2689,25 @@ In the following tables, features are marked with the following statuses:
 |Technology Preview
 |Technology Preview
 |Technology Preview
-// Needs to move to GA after Nov 15, 2023
+
 |Hosted control planes for {product-title} on bare metal
 |Technology Preview
 |Technology Preview
-|Technology Preview
-// Needs to move to GA after Nov 15, 2023
+|General Availability
+
 |Hosted control planes for {product-title} on {VirtProductName}
 |Not Available
 |Technology Preview
+|General Availability
+
+|Hosted control planes for {product-title} on {ibmpowerProductName}
```

Comment:
This attribute is not rendering in the preview. Is it being defined in a different PR that hasn't merged yet?

![image](https://github.com/openshift/openshift-docs/assets/86380613/3f2f8c37-fc6c-4e9a-93b0-6197618bba65)